### PR TITLE
fix cypress tests

### PR DESCRIPTION
**Pull request by ETLaurent**
## Summary

see code comments

(no changelog because fixing stuff that's not been released)

cf https://github.com/apostrophecms/testbed/pull/248

## What are the specific steps to test this change?

*For example:*
> 1. Run the website and log in as an admin
> 2. Open a piece manager modal and select several pieces
> 3. Click the "Archive" button on the top left of the manager and confirm that it should proceed
> 4. Check that all pieces have been archived properly

## What kind of change does this PR introduce?
*(Check at least one)*

- [ ] Bug fix
- [ ] New feature
- [ ] Refactor
- [ ] Documentation
- [ ] Build-related changes
- [ ] Other

## Make sure the PR fulfills these requirements:

- [ ] It includes a) the existing issue ID being resolved, b) a convincing reason for adding this feature, or c) a clear description of the bug it resolves
- [ ] The changelog is updated
- [ ] Related documentation has been updated
- [ ] Related tests have been updated

If adding a new feature without an already open issue, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**

### CODE REVIEW

File: modules/@apostrophecms/widget-type/ui/apos/components/AposWidgetEditor.vue [hunk 1]
```diff
@@ -200,9 +200,9 @@ export default {
       this.evaluateConditions();
     },
     async save() {
-      const widget = klona(this.docFields.data);
       this.triggerValidation = true;
       this.$nextTick(async () => {
+        const widget = klona(this.docFields.data);
```
Comment (ETLaurent) [inline modules/@apostrophecms/widget-type/ui/apos/components/AposWidgetEditor.vue:205]: fixes regression from 8115cdd54d6ad8ac1b28181f0d798a7182010b12

File: modules/@apostrophecms/modal/ui/apos/components/AposWidgetModalTabs.vue
Comment (ETLaurent) [inline modules/@apostrophecms/modal/ui/apos/components/AposWidgetModalTabs.vue:1]: move the logic from AposModalTabs before reverting it to a working state

fixed regression in 565424eada93737ea2fcce963b31dd7eb92b6a31

**Comment by ETLaurent** [comment]
tests run: https://github.com/apostrophecms/testbed/actions/runs/9019485122

File: modules/@apostrophecms/modal/ui/apos/components/AposModalTabs.vue
Comment (ETLaurent) [inline modules/@apostrophecms/modal/ui/apos/components/AposModalTabs.vue:1]: revert to its working state

Comment (ETLaurent) [reply modules/@apostrophecms/modal/ui/apos/components/AposModalTabs.vue:1]: solution made in a61a4edce8dbd27d17c265eb8a61be2d31d4da5e was not enough, we still had an issue with template library.

seems like we're losing  the tabs reactivity... 

the choice to revert this file and create a separate AposWidgetModalTabs.vue file, only used by AposWidgetEditor.vue is the safest one, and fixes all the regressions

**Review by boutell — APPROVED** [review]

File: modules/@apostrophecms/widget-type/ui/apos/components/AposWidgetEditor.vue (hunk 1 above)
Comment (ETLaurent) [reply modules/@apostrophecms/widget-type/ui/apos/components/AposWidgetEditor.vue:205]: Screenshot 2024-05-09 at 13 20 32
Screenshot 2024-05-09 at 13 21 14

**Pull request merged**